### PR TITLE
Clippy fixes v5

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
None 

Describe changes:
- rust: fix clippy warnings in tests

#10129 with commit reworded

A newt PR could make `cargo clippy` clean (without the --all-features, ie the `SCLogDebug`calls)

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 17482

**Review by jasonish — APPROVED** [review]
LGTM.

File: rust/Cargo.lock.in [hunk 1]
```diff
@@ -89,8 +89,8 @@ version = "0.4.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "726535892e8eae7e70657b4c8ea93d26b8553afb1ce617caee529ef96d7dee6c"
 dependencies = [
- "proc-macro2 1.0.69",
- "quote 1.0.33",
+ "proc-macro2 1.0.75",
```
Comment (victorjulien) [inline rust/Cargo.lock.in:92]: this appears to update more than the commit message states. If I update just the test-case crate in the Cargo.toml.in and rebuild, I get a smaller diff. How was this updated?

Comment (catenacyber) [reply rust/Cargo.lock.in:92]: `make update-lock` in rust directory

Looking at the Makefile, I see how to make the small update to Cargo.lock.in

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata/pull/10156

File: rust/Cargo.lock.in (hunk 1 above)
Comment (victorjulien) [reply rust/Cargo.lock.in:92]: I'm fine either way, as long as the message reflects what has been done.

Comment (catenacyber) [reply rust/Cargo.lock.in:92]: I thought it did

Comment (catenacyber) [reply rust/Cargo.lock.in:92]: Now I know and I will try to remember

**Pull request closed without merging**